Readme.md

```diff
@@ -109,7 +110,15 @@ docker run --rm -it --workdir /code -v "$PWD":/code wasm-imagemagick-build-tools
 
 #windows cmd
 #docker run --rm -it --workdir /code -v %CD%:/code wasm-imagemagick-build-tools bash ./build.sh
-
 ```
 
-Produces `magick.js` & `magick.wasm` in the current folder. Along with `magickApi.js` that is all the files needed on the webserver.

```

Comment:
Please add how & where magickApi.js is produced. Is it in /src after running npm run build?
also can you update the publish code to include this
https://github.com/KnicKnic/WASM-ImageMagick/blob/ccecb8bfb9e531f0ee3c3932e97c6a94bdba14e0/azure-pipelines.yml#L25-L26
if you look two blocks up you in the azure pipelines file you can see how to do multiple statements.

Reply:
Also I think your tests will fail till you use npm install here
https://github.com/KnicKnic/WASM-ImageMagick/blob/ccecb8bfb9e531f0ee3c3932e97c6a94bdba14e0/azure-pipelines.yml#L14-L15

Reply:
Didn't support to azure-pipelines CI yet - npm test command is the same as before.  - first wanted to see if this is acceptable to you and review - then will try to make the CI to work. thanks (still pushing changes)

Reply:
> Please add how & where magickApi.js is produced. Is it in /src after running npm run build?
> also can you update the publish code to include this

you are right - magickApi.js that can be loaded in the browser directly is no longer supported. I must compile src/magickApi.ts separately / with different configuration so it's generated.

Reply:
> Didn't support to azure-pipelines CI yet - npm test command is the same as before. - first wanted to see if this is acceptable to you and review - then will try to make the CI to work. thanks (still pushing changes)

BTW I am very happy with what you have done. I am okay with the set of changes in general. If you do not want to learn another CI I can do that work.